kedro-airflow/kedro_airflow/plugin.py

```diff
@@ -180,6 +190,10 @@ def create(  # noqa: PLR0913
         dag_name += "_dag.py"
         dag_filename = dags_folder / dag_name
 
+        if tags:
+            tags_list = tags.split(",")
+            pipeline = pipeline.only_nodes_with_tags(*tags_list)  # noqa: PLW2901
+
         # group memory nodes
```

Comment:
I suspect this is not consistent with `kedro run` or break with some edge cases. I think we can either import from `kedro` if it's not added dependencies or we can just copy paste the parsing logic.

We do `split_string`, which is roughly:
```python

def split_string(ctx: click.Context, param: Any, value: str) -> list[str]:
    """Split string by comma."""
    return [item.strip() for item in value.split(",") if item.strip()]
```

This logic should fails with whitespace (I don't have a test project, please try it)

Reply:
Yes, that's true. Thank you for pointing that out. It was already a dependency, so I added `split_string` to the imports and the callback.